digAgeExtractor/ages.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
it's necessary for now since we need conduct some experiments based on these data and script to improve performance,:)